### PR TITLE
bf: S3C-1286 increase bucket name length

### PR DESCRIPTION
**Pull request by dora-korpar**
Increases max bucket name length to 255 characters from 63.

### CODE REVIEW

File: tests/unit/utils.js [hunk 1]
```diff
@@ -121,9 +121,8 @@ describe('utils.isValidBucketName', () => {
     });
 
     it('should return false if bucketname is greater than ' +
-        '63 characters long', () => {
-        const longString = 'aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa' +
-            'aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa';
+        '255 characters long', () => {
+        const longString = 'a'.repeat(256);
```
Comment (miniscruff) [inline tests/unit/utils.js:125]: yes thanks for not typing out 'a' 256 times :100:

**Review by miniscruff — APPROVED** [review]

**Comment by dora-korpar** [comment]
@bert-e reset

File: lib/utils.js [hunk 2]
```diff
@@ -167,7 +167,7 @@ utils.isValidBucketName = function isValidBucketName(bucketname) {
     const ipAddressRegex = new RegExp(/^(\d+\.){3}\d+$/);
     const dnsRegex = new RegExp(/^[a-z0-9]+([\.\-]{1}[a-z0-9]+)*$/);
     // Must be at least 3 and no more than 63 characters long.
```
Comment (ilkescality) [inline lib/utils.js:169]:
```suggestion
    // Must be at least 3 and no more than 255 characters long.
```

**Review by ilkescality — APPROVED** [review]

**Comment by terrydervaux** [comment]
Sorry @dora-korpar,  bucket name >63 characters is not inline with AWS specifications.

**Pull request closed without merging**